planner/explain.go

```diff
@@ -56,14 +56,114 @@ const (
 	dataLabel           = "data"
 	fieldNameLabel      = "fieldName"
 	filterLabel         = "filter"
-	keysLabel           = "_keys"
 	idsLabel            = "ids"
+	joinRootLabel       = "root"
+	joinSubTypeLabel    = "subType"
+	keysLabel           = "_keys"
 	limitLabel          = "limit"
 	offsetLabel         = "offset"
 	sourcesLabel        = "sources"
 	spansLabel          = "spans"
```

Comment:
suggestion (out of scope): These are all public, and I think they should live in `client/request/explain.go` - might be worth adding a ticket if you agree.

Same also goes for the attribute tags and node names (if they are in planner atm)

Reply:
I appreciate this suggestion! Would be much nicer. Will open a ticket.

Reply:
ticket: https://github.com/sourcenetwork/defradb/issues/1571